langroid/agent/special/sql/sql_chat_agent.py

```diff
@@ -200,19 +234,39 @@ def agent_response(
         )
 
     def retry_query(self, e: Exception, query: str) -> str:
```

Comment:
This does not seem to actually retry the query?

Reply:
should we add a test that exercises retry_query?

Reply:
Never mind, this is not actually meant to retry the query, just sends the err msg so it reformulates query on next round.